Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -78,6 +81,10 @@ public override void Initialize()
         Subs.CVar(_configurationManager, CCVars.LoocEnabled, OnLoocEnabledChanged, true);
         Subs.CVar(_configurationManager, CCVars.DeadLoocEnabled, OnDeadLoocEnabledChanged, true);
         Subs.CVar(_configurationManager, CCVars.CritLoocEnabled, OnCritLoocEnabledChanged, true);
+        Subs.CVar(_configurationManager, CCVars.DiscordLastMessageBeforeDeathWebhook, value =>
+        {
+            if (!string.IsNullOrWhiteSpace(value)) _lastMessageWebhookEnabled = true;
+        }, true);
```

Comment:
This boolean should be tracked fully in the other system, have `LastMessageBeforeDeathSystem` decide whether to ignore it or not.